go/store/nbs/gc_copier.go

```diff
@@ -92,13 +93,28 @@ func (gcc *gcCopier) copyTablesToDir(ctx context.Context, tfp tableFilePersister
 		}, nil
 	}
 
+	// Attempt to rename the file to the destination if we are working with a fsTablePersister...
+	if fstp, ok := tfp.(*fsTablePersister); ok {
```

Comment:
Probably better to have an interface for this set of requirements / capabilities